gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java

```diff
@@ -82,8 +82,8 @@ public  void generateSummaryEvents(DichotomyDirection direction, DichotomyResult
             limitingElement = DichotomyResultHelper.getLimitingElement(crac, raoResult);
             printablePrasIds = toString(DichotomyResultHelper.getActivatedActionInPreventive(crac, raoResult));
             printableCrasIds = toString(DichotomyResultHelper.getActivatedActionInCurative(crac, raoResult));
-            if (dichotomyResult.getHighestValidStep().getValidationData() != null && dichotomyResult.getHighestValidStep().getValidationData().getAngleMonitoringResult() != null) {
-                angleCheckStatus = dichotomyResult.getHighestValidStep().getValidationData().getAngleMonitoringResult().getStatus().name();
+            if (dichotomyResult.getHighestValidStep().getValidationData() != null) {
+                angleCheckStatus = dichotomyResult.getHighestValidStep().getValidationData().getAngleMonitoringResult().name();
```

Comment:
Are we 100% sure that angleMonitoringStatus will never be null? If no, add a not-null check before calling `.name()`.

Reply:
Attribute is final and constructor is always supplied with an enum != null, hence it cannot be null. I'll add a not-null check just to be extra cautious